packages/edit-site/src/components/posts-app/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	UnsavedChangesWarning,
+	privateApis as editorPrivateApis,
+} from '@wordpress/editor';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import Layout from '../layout';
+import Page from '../page';
+import { unlock } from '../../lock-unlock';
+
+const { RouterProvider } = unlock( routerPrivateApis );
+const { GlobalStylesProvider } = unlock( editorPrivateApis );
+
+const defaultRoute = {
+	key: 'index',
+	areas: {
+		sidebar: 'Empty Sidebar',
+		content: <Page>Welcome to Posts</Page>,
+		preview: undefined,
+		mobile: <Page>Welcome to Posts</Page>,
+	},
+};
```

Comment:
@jsnajdr I wish we can declare the routes of the site editor in a static way like this with an extra "path" or something.

For now, having a useLayoutArea hook for the site editor and another one for this new experimental page... works. But there's something to be done here.

Reply:
We'll need to have a hook similar to `useLayoutAreas` here to get params from location later, right?

Reply:
Yes